include/eosio/to_key.hpp

```diff
@@ -111,6 +111,12 @@ void to_key(const std::vector<T>& obj, S& stream) {
    to_key_optional((const T*)nullptr, stream);
 }
 
+template <typename S>
+void to_key(const std::vector<bool>& obj, S& stream) {
+   for (bool elem : obj) { to_key_optional(&elem, stream); }
+   to_key_optional((const bool*)nullptr, stream);
+}
+
```

Comment:
Where in the code the warning complains?

Reply:
Where ever this method is invoked with a vector of booleans.